src/sip/transp.c

```diff
@@ -47,6 +47,12 @@ struct sip_ccert {
 };
 
 
+struct ccert_data {
```

Comment:
Perhaps add `sip_` prefix to match the name of the module ?

Reply:
Done.